cypress/support/helper.js

```diff
@@ -28,17 +29,29 @@ export function visitComponentUrlWithParameters(
   const onBeforeLoad = cy.spy();
   cy.fixture(`${path}/${json}`).then(($json) => {
     const el = $json[nameOfObject];
-    let url = "&args=";
+    let url = "";
+    let args = [];
+    let globals = [];
     for (const prop in el) {
       if (prop === "theme") {
-        url += `&theme=${encodeURIComponent(el[prop])}`;
+        globals.push(`${prop}:${encodeURIComponent(el[prop])}`);
```

Comment:
As you're pushing the same string onto different arrays here, couldn't you just concat them onto the url string in the for loop instead of having the extra conditionals below?

eg:

```
url += `&${prop === "theme" ? "globals" : "args"}=${prop}:${encodeURIComponent(el[prop])}`
```

Reply:
Discussed on slack and I'm happy to leave this as is